aerosol_dynamics/physical_parameters.py

```diff
@@ -11,7 +11,7 @@
 MEAN_FREE_PATH_AIR = 65e-9 * u.m
 
 # Boltzmann's constant in m^2 kg s^-2 K^-1
-BOLTZMANN_CONSTANT = 1.380649e-23 * u.m ** 2 * u.kg / u.s / u.K
+BOLTZMANN_CONSTANT = 1.380649e-23 * u.m ** 2 * u.kg / (u.s**2) / u.K

```

Comment:
good catch

Reply:
The beauty of having unit checks everywhere!

Reply:
I should have pushed out a hotfix for that, sorry! I caught it while working on the Environment class (yes, units checks!) but thought I'd be able to get it out faster.

Reply:
@wkchuang that's why it is good to do smaller commits on the fly like I do --- your work won't be hidden from us. Our PRs ad commits don't have to be perfect, as long as we are getting stuff done. Public mistakes are fine --- after all, that's the main way I've been learning all of this!